mitxgraders/helpers/calc.py

```diff
@@ -29,48 +26,44 @@
     alphas,
     nums,
     stringEnd,
-    ParseException
+    ParseException,
+    delimitedList
```

Comment:
sweet :)

Reply:
I know, right? I was quite happy not to have to construct it in terms of more low-level functionality. And it will make doing vectors/matrices quite straightforward, too!